### ISSUE TITLE
Suspense Mode

### ISSUE THREAD

**Issue by rotexhawk** [zeit/swr#58]
I noticed two problems with the suspense mode. 

1. The default example `useSWR('/api', fetch, { suspense: true })` doesn't work because it returns a readable stream. 

2. When I modified the default example and returned data, I noticed, the component was rendered multiple times. 

```javascript

import React from 'react';
import useSWR from 'swr'

const FetchSuspense = endpoint => {
    console.log('calling me with', endpoint)
    return fetch(endpoint).then(data => data.json()).then(res => res.results[0])
}

export const Person = ({ resource, num  }) => {
     const { data } = useSWR('https://randomuser.me/api', FetchSuspense, { suspense: true })
     console.log("calling comp", num)
     return (<div>Loaded {num}: - {data.name.first}</div>)
};


```

**Comment by icyJoseph** [comment]
Yes, I think the docs might be misleading when using the word fetch, as you are supposed to make your own handler, as oppose to pass window.fetch.

**Comment by quietshu** [comment]
Thank you! We'll make the documentation about `fetch` clearer.

The second question should be related to #57.

**Comment by quietshu** [comment]
Shoud be fixed in `swr@0.1.8`.
